backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/RabbitMQManagementAPI.java

```diff
@@ -59,6 +59,9 @@
         @JsonProperty("arguments")
         Map<String, String> arguments;
 
+        @JsonProperty("messages")
+        Long messages;
```

Comment:
I prefer to rename this field to `messageCount` while keeping the `@JsonProperty("messages")`